### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by trezy**
The docs were bland and tasteless. Now they have flavor and they don't make anybody use superfluous aliases!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 npm install
 ```
 
-### Running things
+Then to start the build pipeline and run the server:
+
 ```shell
 # starts the app in the browser with browsersync
-$(npm bin)/gulp serve
+npm start
```
Comment (rylnd) [inline README.md]: This should be explicit and consistent with the deploy script:

```bash
npm run start
```

**Comment by rylnd** [comment]
What aliases were we making people use?

**Comment by rylnd** [comment]
The documentation does not instruct you to use `ne`. Let's please keep the PR notes positive and grounded in reality.

**Comment by rylnd** [comment]
Of course. I'm just trying to keep the conversation relevant. Aliases created/used outside the documentation are not "under warranty," as it were.

**Comment by trezy** [comment]
@rylnd The original alias was `$(npm bin)/gulp serve`, which just uses the gulp binary in the local node bin folder. The new docs use the `npm start` script which is a standard npm alias for starting an application. The benefit is that we can now change the script to run the application -- for example, if we switched to Grunt or Broccoli instead of Gulp -- but the docs remain the same.

As for maintaining consistency with `npm script` vs `npm run script`, `npm start` is a standard script. However, the scripts object can be extended with any number of non standard script names. They just need to be used with `npm run script`. Basically, `npm run` isn't required for the start script, it is required for the deploy script.

**Comment by rylnd** [comment]
I was approaching it from the perspective of "these are both user scripts defined within lines of each other, yet we run them differently." I forgot about the builtins. 👍

**Pull request merged**